includes/plugins/class-newspack-newsletters.php

```diff
@@ -54,6 +28,68 @@ public static function init() {
 		}
 	}
 
+	/**
+	 * Fetch the prefix for synced metadata fields.
+	 * Default is NP_ but it can be configured in the Reader Activation settings page.
+	 *
+	 * @return string
+	 */
+	public static function get_metadata_prefix() {
+		$prefix = \get_option( self::METADATA_PREFIX_OPTION, self::METADATA_PREFIX );
+
+		// Guard against empty strings and falsy values.
+		if ( empty( $prefix ) ) {
+			return self::METADATA_PREFIX;
+		}
+
+		return $prefix;
```

Comment:
It can be useful to have this filterable.

Reply:
Good idea. Done in 3659d6e!